### DISCUSSION TITLE
Is this the file that needs to be updated for settings page text?

### DISCUSSION THREAD

**Discussion by littleboy0k** [ecency/ecency-vision#648] [Q&A] [answered]
There are some grammar errors and potential improvements I have in mind. Is this the file that needs to be updated for settings page text? 

https://github.com/ecency/ecency-vision/blob/ffbff8e83bd5c546b259d99650a03d839f98a77a/src/common/components/wallet-hive/__snapshots__/index.spec.tsx.snap

**Comment by feruzm** [comment, marked as answer]
@littleboy0k For grammar errors, you will need to update translation files. Submit Pull request with changes on this file: https://github.com/ecency/ecency-vision/blob/development/src/common/i18n/locales/en-US.json
